pages/book/structs-and-messages.mdx

```diff
@@ -209,9 +209,89 @@ fun conversionFun() {
 <Callout>
 
   See those extension functions in the Reference:\
-  [`Struct.toCell(){:tact}`](/ref/api-cells#structtocell)\
-  [`Message.toCell(){:tact}`](/ref/api-cells#messagetocell)
+  [`Struct.toCell(){:tact}`](/ref/core-cells#structtocell)\
+  [`Message.toCell(){:tact}`](/ref/core-cells#messagetocell)
 
 </Callout>
 
+### Obtain from a `Cell` or `Slice`, `.fromCell()` and `.fromSlice()` [#fromcellslice]
+
+Instead of manually parsing a [`Cell{:tact}`][p] or [`Slice{:tact}`][p] via the series of relevant `.loadSomething(){:tact}` function calls, one can use `.fromCell(){:tact}` and `.fromSlice(){:tact}` [extension functions](/book/functions#extension-function) for converting the provided [`Cell{:tact}`][p] or [`Slice{:tact}`][p] into the needed [Struct](#structs) or [Message](#messages).
```

Comment:
```suggestion
Instead of manually parsing a [`Cell{:tact}`][p] or [`Slice{:tact}`][p] via a series of relevant `.loadSomething(){:tact}` function calls, one can use `.fromCell(){:tact}` and `.fromSlice(){:tact}` [extension functions](/book/functions#extension-function) for converting the provided [`Cell{:tact}`][p] or [`Slice{:tact}`][p] into the needed [Struct](#structs) or [Message](#messages).
```